sql/hive/src/test/scala/org/apache/spark/sql/hive/CachedTableSuite.scala

```diff
@@ -421,7 +421,7 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
           activateDatabase(db) {
             sql("REFRESH TABLE default.cachedTable")
             assertCached(
-              sql("SELECT * FROM default.cachedTable"), "`default`.`cachedTable`", DISK_ONLY)
+              sql("SELECT * FROM default.cachedTable"), "cachedTable", DISK_ONLY)
```

Comment:
@cloud-fan Looks like we need to resolve catalog/current namespace for this scenario?

Reply:
I guess this is OK as long as `spark.catalog.isCached` works fine since it's just a name of cache builder?

Reply:
Why is `default.` gone here?

Reply:
Because the cache was created with `CACHE TABLE cachedTable OPTIONS('storageLevel' 'DISK_ONLY')`. Before, `cachedTable` was resolved to `default.cachedTable` in `ResolveSessionCatalog`.

Note that it's just a name of cache builder and doesn't affect `spark.catalog.isCached(tableName: String)`.